fuel/src/test/kotlin/com/github/kittinunf/fuel/core/HeadersTest.kt

```diff
@@ -5,7 +5,7 @@ import org.hamcrest.CoreMatchers.not
 import org.hamcrest.CoreMatchers.notNullValue
 import org.hamcrest.MatcherAssert.assertThat
 import org.junit.Test
-import org.hamcrest.CoreMatchers.`is` as isEqualTo
+import org.hamcrest.CoreMatchers.`is` as equalTo
```

Comment:
[Nits] I think we can just use `equalsTo()` from the `hamcrest` itself.

Reply:
Yep! Was supposed to be that.